.evergreen/download-mongodb.sh

```diff
@@ -57,11 +57,11 @@ get_mongodb_download_url_for ()
    VERSION_RAPID="6.3.1"
    VERSION_70="7.0.0-rc3"
```

Comment:
Would you mind updating this to r7.0.0-rc7 as well?

Reply:
Sure thing! Would that be `r7.0.0-rc7`, or `7.0.0-rc7` without the leading `r`?

Reply:
7.0.0-rc7 without the leading r

Reply:
Cool, feel free to merge when you can.